client/js/components/week/week-event.js

```diff
@@ -0,0 +1,20 @@
+/** @jsx React.DOM */
+var React = require('react');
+var _ = require('../../utils/utils');
+
+var WeekEvent = React.createClass({
+  render: function(){
+    return (
+      <div>
+        <h5>{ this.props.data.title }</h5>
+        <p>{ this.props.data.start }</p>
+        <p>{ this.props.data.end }</p>
+        <p>{ this.props.data.health  }</p>
+        <p>{ this.props.data.totalStaffed[0] }</p>
```

Comment:
what's the difference between totalStaffed[0] and totalStaffed[1]? it be cool if you included a JSON object with this commit to see what data you're representing